aio_pika/connection.py

```diff
@@ -387,8 +387,8 @@ def connect(url: str=None, *, host: str='localhost',
 
     ssl_options = ssl_options or {}
 
-    if url:
-        url = URL(str(url))
+    if url is not None:
+        url = URL(url)
```

Comment:
This operation makes once and the `URL(str(url))` is more comfortable. Yes it's not fast and ugly, but so useful.

Reply:
casting to str is redundant and removing it wouldn't break compatibilty because yarl.URL can take another object of this type in the constructor and will simply return a copy of it